rest/README.md

```diff
@@ -1,2 +1,39 @@
 # Discord4J Rest
-This module contains low level abstractions pertaining to the Discord API rest endpoints.
+The `rest` module provides a low-level HTTP client specifically for Discord which properly handles Discord's [ratelimiting system](https://discordapp.com/developers/docs/topics/rate-limits).
+
+## Installation
+```groovy
+repositories {
+  maven { url  "https://jitpack.io" }
+}
+
+dependencies {
+  implementation "com.discord4j.discord4j:discord4j-rest:@VERSION@"
+}
+```
+
+## Example Usage
+```java
+final ObjectMapper mapper = new ObjectMapper()
+                .setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
+                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true)
+                .registerModules(new PossibleModule(), new Jdk8Module());
+
+final SimpleHttpClient httpClient = SimpleHttpClient.builder()
+                .defaultHeader("content-type", "application/json")
+                .defaultHeader("authorization", "Bot " + token)
+                .defaultHeader("user-agent", "DiscordBot(" + url + ", " + version + ")")
```

Comment:
I think we should encourage the inclusion of d4j on there in case b1nzy tracks usage again

Reply:
I don't think we can really control it. Anyone can change it, or not include it at all. This is just an example.

Reply:
I understand that, but iti's what a majority of people would base their code on

Reply:
Im not sure how best to do it. I dont think it's a good idea to include the version considering we'd have to update it.

Reply:
I think the psuedocode-ish version variable is fine.